src/main/java/Cleese/TaskList.java

```diff
@@ -0,0 +1,35 @@
+package Cleese;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private final ArrayList<Task> taskList;
+
+    public TaskList() {
+        this.taskList = new ArrayList<>();
+    }
+    public void printTaskList() {
+        for (int i = 0; i < taskList.size(); i++) {
+            System.out.print(String.format("%d. %s \n", i+1, taskList.get(i).toString()));
+        }
+    }
+    public boolean removeFromTaskList(Task selectedTaskToDelete) {
+        return taskList.remove(selectedTaskToDelete);
+    }
+    public Task get(Integer i) {
```

Comment:
Might want to rename the function instead to be more descriptive.
```suggestion
    public Task getTask(Integer i) {
```